### PR TITLE
feat(dashboard): Allow dashboards to use external etcd with mtls

### PR DESCRIPTION
**Pull request by jaysonsantos**
Example values would be:

```yaml
config:
  conf:
    etcd:
      endpoints:
        - https://my-cname.domain.com:2379
      mtls:
        key_file: /etc/etcd/server-client.key
        cert_file: /etc/etcd/server-client.crt
        ca_file: /etc/etcd/server-ca.crt
      existingSecret: cluster-etcd-credentials
```

### CODE REVIEW

**Review by tao12345666333 — COMMENTED** [review]
LGTM, Thanks!


Please update the README documents also.

**Comment by jaysonsantos** [comment]
@tao12345666333 I forgot about this, I just rebased and squashed the old commits into one, can you take a look?

**Comment by tao12345666333** [comment]
Thanks, wait for CI

**Comment by jaysonsantos** [comment]
@tao12345666333 i ran locally helm-docs on that folder to regenerate the readme

File: charts/apisix-dashboard/values.yaml [hunk 1]
```diff
@@ -89,6 +89,16 @@ config:
       username: ~
       # -- Specifies etcd basic auth password if enable etcd auth
       password: ~
+
+      # -- Specifies a secret to be mounted on /etc/etcd for mtls usage
+      existingSecret: ""
```
Comment (tao12345666333) [inline charts/apisix-dashboard/values.yaml:94]: I think this name is ambiguous. It would be better to add mtls to it.

Comment (jaysonsantos) [reply charts/apisix-dashboard/values.yaml]: done

**Review by tao12345666333 — APPROVED** [review]

**Review by Gallardot — APPROVED** [review]

**Pull request merged**